frontend/rust-lib/flowy-database2/src/services/field/type_options/media_type_option/media_type_option.rs

```diff
@@ -132,7 +126,7 @@ impl TypeOptionCellDataCompare for MediaTypeOption {
     other_cell_data: &<Self as TypeOption>::CellData,
     _sort_condition: SortCondition,
   ) -> Ordering {
-    match (cell_data.files.is_empty(), other_cell_data.is_cell_empty()) {
+    match (cell_data.files.is_empty(), other_cell_data.is_empty()) {
```

Comment:
unrelated. Can this directly use the `is_empty` check?

Reply:
Nice catch! One test was failed because of this. I've fixed it

Reply:
Oh cool!